### ISSUE TITLE
Voted Ticket Notification

### ISSUE THREAD

**Issue by pedrotst** [decred/dcrstakepool#201]
[Feature Request]
It would be nice to have some sort of notification system to notify when a ticket was voted. This way we don't have to check the pool website everyday to check that. Specially now that we don't have buy windows anymore.

**Comment by jolan** [comment]
A couple of the pools have this feature implemented already.  For the reference implementation, maybe another standalone program that could live in backend.

**Comment by jolan** [comment]
A couple of the pools have this feature implemented already.  For the reference implementation, maybe another standalone program that could live in backend.

**Comment by chappjc** [comment]
@jolan Yeah, that sounds like a good idea to keep it separate.  Just a little app that waits for voted notifications, looks up the email for the redeemed ticket multisig, and send an email.  It's the part where it looks up emails that I'm not so sure about since what we ultimately want to for MySQL to be untouched by anything but dcrstakepool

**Comment by jolan** [comment]
This is needed so we can mark low fee tickets as voted in the database (continuing on #219) and also so we can remove any reference to stakepooluserinfo on the voting page / getstakeinfo on the stats page (#180).

**Comment by xaur** [comment]
Please add 'privacy' label. Notifications have a privacy aspect: the service that you receive notification to (e.g. your email provider) will learn about your holdings and voting preferences. There should be a way to encrypt notifications to recipient's public key.

**Comment by kiahmed** [comment]
have this ever been implemented or any script that could be hooked up?
